### ISSUE TITLE
NTR: Muscle biopsy: caveolin low or absent (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9112]
New term request Muscle biopsy: caveolin low or absent (segal_090126230502):   UMLS: 1843059

**Comment by pnrobinson** [comment]
@MickeySegal
would this be calveolin-1? e.g., 
https://www.abcam.com/products/primary-antibodies/alexa-fluor-488-caveolin-1-antibody-e249-caveolae-marker-ab185043.html

**Comment by MickeySegal** [comment]
We have it for https://www.omim.org/entry/606072 and https://www.omim.org/entry/614321 and it is caveolin-3.  Also in https://www.ncbi.nlm.nih.gov/medgen/C1843059 and https://www.ncbi.nlm.nih.gov/medgen/C3280446.

I've added the 3 to our name.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000230
- label: Decreased muscle caveolin-3 level
- parents: [Abnormal muscle tissue metabolite concentration (HP:0034370)](https://hpo.jax.org/app/browse/term/HP:0034370)
- pmid: PMID:12666119
- def: The amount of calveolin-3 in muscle tissue is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9112
- synonyms: Decreased muscle caveolin-3 expression(NONE)
